shell/scl_source

```diff
@@ -7,7 +7,7 @@ Don't use this script outside of SCL scriptlets!
 Options:
     -h, --help    display this help and exit"
 
-if [ $# -eq 0 -o $1 = "-h" -o $1 = "--help" ]; then
+if [ $# -eq 0 -o X"$1" = X"-h" -o X"$1" = X"--help" ]; then
```

Comment:
One could argue that displaying help "did nothing" and therefore failed as well. Just pushed that scenario.